src/conn/api_strerror.c

```diff
@@ -43,6 +41,19 @@ __wt_wiredtiger_error(int error)
         return ("WT_TRY_SALVAGE: database corruption detected");
     }
 
+    /* Check for WiredTiger specific sub-level errors. */
+    switch (error) {
+    case WT_NONE:
+        return ("WT_NONE: last API call was successful");
```

Comment:
Just checking, in the Tech Design the error message string consists of only the text after the colon, is that accurate or should it include the name of the error code as well (as is returned here)?

Reply:
Great question: it should include the error name as well, in a similar way to how it is done for WT error codes. Eg
`return ("WT_ROLLBACK: conflict between concurrent operations");`

Reply:
I see, makes sense thanks!